docs/management/admin/admin-pg-ov.asciidoc

```diff
@@ -87,17 +87,17 @@ image::images/advanced-settings.png[Integration page]
 
 The status of the policy application appears in the *Policy Status* column and displays one of the following possibilities:
 
-* *Success:* The policy applied successfully.
+* `Success`: The policy was applied successfully.
 
-* *Warning or Partially Applied:* The policy is pending application, or the policy was not applied in its entirety.
+* `Warning or Partially Applied`: The policy is pending application, or the policy was not applied in its entirety.
 
 NOTE: In some cases, some actions taken on the endpoint may fail during the policy application but are not recognized as a critical failure - meaning there may be a failure, but the endpoints are still protected. In this case, the policy status will display as "Partially Applied."
 
-* *Failure:* The policy did not apply correctly. As such, endpoints are not protected.
+* `Failure`: The policy did not apply correctly. As such, endpoints are not protected.
 
-* *Unknown:* The user interface is waiting for the API response to return, or, in rare cases, the API returns an undefined error or value.
+* `Unknown`: The user interface is waiting for the API response to return, or, in rare cases, the API returns an undefined error or value.
 
-To view policy status details, click the link and review the data in the flyout panel. In the following image, you can see that the malware configuration and logging failed, generating a "Failed" policy status.
+To view policy status details, click the link and review the data in the flyout that displays. In the following image, you can see that the malware configuration and logging failed, generating a "Failed" policy status.
```

Comment:
Looking at the image, it's actually not immediately clear that malware config and logging failed, as this text claims. The image has "Success" on all the statuses listed, and the **Malware** and **Logging** sections are collapsed so we don't see any statuses for them.

Reply:
I had updated this image, so good eye! Will delete this sentence.